terraform/services/github-actions-role/data.tf

```
@@ -23,13 +23,20 @@ locals {
       "web-admin",
     ] : [],
   )
+
+  sops_env = contains(["dev", "test"], var.env) ? "test" : "prod"
```

Let's generalize this a bit more for other resources


```suggestion
  # TODO Replace with cdap-test and cdap-prod when those environments are set up
  account_env = contains(["dev", "test"], var.env) ? "bcda-test" : "bcda-prod"
```